README.md

```diff
@@ -46,33 +46,57 @@ Response from `https://liz-taylor-palette.herokuapp.com/api/v1/projects/1` examp
 ]
 ```
 
-#### POST `/api/v1/projects/` (Create New Project)
-A user can add an additional project the database. Below is the required parameters and an example post.
+If no `id` matches the parameter in the request, the response sends an error message with a 404 status code. 
+
+Response from `https://liz-taylor-palette.herokuapp.com/api/v1/projects/999` example:
+```json
+[
+  {
+    "error": "Project with id 999 not found."
+  }
+]
+```
 
+#### POST `/api/v1/projects/` (Create New Project)
+A user can add an additional project to the database with the following required parameters:
 | Name         | type      | Description                             |
 | :------------|:----------|:----------------------------------------|
 | name         | string    | unique name for each project            |
 
 POST options object example:
-```
+```json
   {
-    method: "PUT",
+    method: "POST",
```

Comment:
Looks like this JSON data wasn't formatted quite right. Let's look at this and see if we can get it corrected.